tools/clang/lib/SPIRV/SpirvEmitter.h

```diff
@@ -593,6 +592,11 @@ class SpirvEmitter : public ASTConsumer {
   /// Emits an error if the given attribute is not a loop attribute.
   spv::LoopControlMask translateLoopAttribute(const Stmt *, const Attr &);
 
+  static hlsl::ShaderModel::Kind
+  SpirvEmitter::getShaderModelKind(StringRef stageName);
```

Comment:
No need for `SpirvEmitter::`

```cpp
static hlsl::ShaderModel::Kind getShaderModelKind(StringRef stageName);
```

This is [caught by buildbots](https://travis-ci.org/Microsoft/DirectXShaderCompiler/builds/495742990?utm_source=github_status&utm_medium=notification) on Linux and macOS
Same for `getSpirvShaderStage` below.